akka-cluster-sharding/src/main/resources/reference.conf

```diff
@@ -24,8 +24,8 @@ akka.cluster.sharding {
   remember-entities = off
 
   # Set this to a time duration to have sharding passivate entities when they have not
-  # gotten any message in this long time. Set to 'off' to disable.
-  passivate-idle-entity-after = off
+  # received any message in this length of time. Set to 'off' to disable.
+  passivate-idle-entity-after = 5 s
```

Comment:
I think the default should be longer. `120 s` is used in Lagom and there has been no complaints about that. You can also steal some more elaborate description of the purpose of this setting from Lagom: https://github.com/lagom/lagom/blob/master/persistence/core/src/main/resources/reference.conf#L27